qa/L0_perf_analyzer_capi/test.sh

```diff
@@ -288,6 +293,18 @@ if [ $(cat $CLIENT_LOG | grep "${NON_SUPPORTED_ERROR_STRING}" | wc -l) -ne 1 ];
 fi
 set -e
 
+# Testing erroneous configuration
+# This model is expected to fail
+$PERF_ANALYZER -v -m bls_undefined --shape INPUT0:1048576 -t 64\
```

Comment:
You need to update copyright in this file too.

Reply:
done